### DISCUSSION TITLE
Can I build a project that on my device?

### DISCUSSION THREAD

**Discussion by normidar** [ethanblake4/dart_eval#77] [General]
Hey friend, I want to create an IDE to edit my dart project.
Use the created IDE I can work on the train, but  on the documents I can't find a way to build a dart project.

Maybe I will copy all my codes to a string and complete it, is there some easier way to do it?

**Comment by ethanblake4** [comment]
Hi, I'm afraid I don't understand. Can you explain more what exactly you are trying to do? Then I will be happy to help.

**Comment by normidar** [comment]
Complete files on runtime.

**Comment by normidar** [comment]
use `path_provider` to get a directory, there are more than one dart files in the folder, how to run them on realtime?

**Comment by ethanblake4** [comment]
Oh gotcha. I have code that does this, you can look at it: https://github.com/ethanblake4/dart_eval/blob/4f98c9a6e2117fbff429f3f2f960ab6cda6869cc/bin/dart_eval.dart#L81-L103

**Comment by normidar** [comment]
Wow, Thank you very much!
